examples/linear_model/plot_sparse_logistic_regression_20newsgroups.py

```diff
@@ -40,7 +40,7 @@
 solver = "saga"
 
 # Turn down for faster run time
-n_samples = 10000
+n_samples = 7000
```

Comment:
I'd even go lower, maybe 4000 would still be enough?

Reply:
I'll look into it, probably 4000 is still enough :)

Reply:
Yes, time for second example is now down to 5 sec (from 9) but accuracy also dropped a bit (from 0.7 to 0.6). Will check out 5000 and 6000

Reply:
Okay, 5000 seems to be a good compromise, time is down to under 7 sec with accurace close to where it was before